test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -174,37 +174,63 @@ session_simulator::commit_transaction(const std::string &config)
     WT_SIM_RET_MSG(ts_manager->parse_config(config, supported_ops, unsupported_ops, config_map),
       "Incorrect config (" + config + ") passed in commit_transaction");
 
+    uint64_t commit_ts = 0;
+    uint64_t durable_ts = 0;
+
+    if (has_first_commit_timestamp())
+        commit_ts = _commit_ts;
+
+    if (is_durable_ts_set())
+        durable_ts = _durable_ts;
+
+    int ret;
     auto pos = config_map.find("commit_timestamp");
```

Comment:
To avoid creating the string two times, we could create a variable for "commit_timestamp".

Reply:
It is only used once, the other one is called commit timestamp.

Reply:
We use the same string when validating the hex value

Reply:
When validating we use `commit timestamp`, and here we use `commit_timestamp`

Reply:
Ok, sorry, I missed that!